metricflow/test/compare_df.py

```diff
@@ -25,20 +48,7 @@ def assert_dataframes_equal(
             sort_by = list(sorted(actual.columns.tolist()))
             expected = expected.loc[:, sort_by].sort_values(sort_by).reset_index(drop=True)
             actual = actual.loc[:, sort_by].sort_values(sort_by).reset_index(drop=True)
```

Comment:
Does it make sense to include an `actual.shape == expected.shape` comparison here?

Reply:
It's already there - line 23. It might be worth making that a little more obvious, I missed it too.

Reply:
Because line 49 will fail if `actual` is bigger than `expected`?

Reply:
Yes, and also if actual is smaller than expected. `equal` is initialized False, and if `expected.shape != actual.shape` none of this code executes. We immediately fall through to throwing the exception.

Reply:
Moved some things around for clarity.